Tribler/Core/APIImplementation/LaunchManyCore.py

```diff
@@ -680,29 +680,35 @@ def early_shutdown(self):
         self.shutdownstarttime = timemod.time()
         if self.boosting_manager:
             yield self.boosting_manager.shutdown()
-            self.boosting_manager = None
+        self.boosting_manager = None
```

Comment:
Because anything can evaluate to False you have to set them to None nonetheless.